mkdocs/mkdocs.yml

```diff
@@ -0,0 +1,43 @@
+site_name: Portal backend docs
```

Comment:
What portal?

Reply:
Fixed that: I now uses mkdocs for docs; because we already have som experience with it.
Not sure if thats a good choice...

Curious if you know some better options